tests/plugins/test_check_and_set_rebuild.py

```diff
@@ -156,4 +157,5 @@ def test_409_response(self, monkeypatch):
                 }
             }
         }))
-        runner.run()
+        with pytest.raises(PluginFailedException):
```

Comment:
Why is it raising an exception now? Sounds like we've changed behavior.

Reply:
The test scenario changed and now no longer correctly tests 409 retries. @rcerven, I think we just need to undo these changes. This retry-on-conflict testing really belongs in osbs-client in any case.

Reply:
Ok I get why it's raising the exception now. Previously this test was testing the retry logic built into the plugin. Since that's gone and handled by `osbs-client`, it's no longer needed to test retry logic here.

The reason this test exists is because the plugin used to care about the 409 status code. It no longer does. I think it provides no value as is. Probably better to remove it, but I'm fine either way.